src/lib/modules/contracts_manager/index.js

```diff
@@ -220,6 +223,27 @@ class ContractsManager {
     );
   }
 
+  _emitContractLogError(address, contractName, functionAbi, inputs, errorMessage){
```

Comment:
Any idea why it doesn't go through the proxy?
Is it only because proxy emits success?

Reply:
The contract method invocation goes through the proxy, but this catches errors that occur during the method invocation, allowing us to log them.